gems/smithy/lib/smithy/views/client/shape_to_hash.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+module Smithy
+  module Views
+    module Client
+      # @api private
+      module ShapeToHash
+        class << self
+
+          def transform_value(model, value, shape)
+            return value unless shape && value
+
+            case shape['type']
```

Comment:
Do we need to cover enum and enumInt here?

Reply:
I don't think we should need to - in the untransformed hash they will just be strings and numbers, which will translate fine.